relay-kafka/src/producer/mod.rs

```diff
@@ -36,6 +39,10 @@ pub enum ClientError {
     #[error("failed to serialize json message")]
     InvalidJson(#[source] serde_json::Error),
 
+    /// Failed to run schema validation on message.
```

Comment:
Can this be conditional?

```suggestion
    #[cfg(debug_assertions)]
    /// Failed to run schema validation on message.
```

Reply:
yes, it has to be, fixed